Tools/CISupport/ews-build/config.json

```diff
@@ -172,6 +172,7 @@
     { "name": "ews228", "platform": "visionos-simulator-1" },
     { "name": "ews229", "platform": "visionos-simulator-1" },
     { "name": "ews230", "platform": "visionos-simulator-1" },
+    { "name": "ews237", "platform": "*" },
```

Comment:
can add this bot separately as part of config changes, since that's kind of unrelated to the core logic.